docs/_articles/en/index.md

```diff
@@ -24,6 +24,8 @@ lang: en
 
 Salesforce Code Analyzer (Code Analyzer) is a unified tool for source code analysis. Code Analyzer analyzes multiple languages. It relies on a consistent command-line interface and produces a results file of rule violations. Use the results to review and improve your code.
 
+If you’re listing a managed package on AppExchange, it must pass security review. You’re also required to upload your Salesforce Code Analyzer scan reports. Attach your Code Analyzer reports to your submission in the AppExchange Security Review Wizard. For more info, read about the [AppExchange Security Review](https://developer.salesforce.com/docs/atlas.en-us.packagingGuide.meta/packagingGuide/security_review_overview.htm) and [Scan Your Code with Salesforce Code Analyzer](https://developer.salesforce.com/docs/atlas.en-us.packagingGuide.meta/packagingGuide/security_review_code_analyzer_scan.htm).
```

Comment:
The text matches what's in the draft doc, but the referenced pages haven't gone live yet so I can't verify that the links work.